cmd/tempo/main.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	"github.com/cortexproject/cortex/pkg/util/flagext"
 	"github.com/cortexproject/cortex/pkg/util/log"
+	"github.com/drone/envsubst"
```

Comment:
nit: These need to be shuffled around. PTAL at - https://github.com/grafana/tempo/blob/main/CONTRIBUTING.md#go-imports

Reply:
I have updated it but I am still not sure if I placed it properly. Also I don't see the guidelines of CONTRIBUTING.md being followed everywhere: eg. https://github.com/grafana/tempo/blob/main/cmd/tempo-cli/main.go

Reply:
Yup, you are correct, we need a step in `make fmt`. Feel free to pick that up if you can :)